.github/workflows/performance-tests.yml

```
@@ -0,0 +1,494 @@
+            ---
+            *Performance tests run on release builds only. See [Performance Testing Guide](../docs/performance/image-optimization-visual-qa.md) for manual QA checklist.*
+            `;
+
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: body
+            });
+
+      - name: Comment PR with Performance Results
```

actionlint (1.7.8)

step must run script with "run" section or run action with "uses" section

(syntax-check)